dev-lang/python-oem/python-oem-3.9.11.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="This project produces self-contained, highly-portable Python distributions."
+HOMEPAGE="https://github.com/indygreg/python-build-standalone/"
+SRC_URI="
+	amd64? ( https://github.com/indygreg/python-build-standalone/releases/download/20220318/cpython-3.9.11+20220318-x86_64_v2-unknown-linux-gnu-install_only.tar.gz )
```

Comment:
These are the binary releases, or? I think it's ok to use binary releases if we have some criteria but at the moment we normally compile from source (also possible for [this project](https://python-build-standalone.readthedocs.io/en/latest/building.html#linux)).
So that's more a general question worth discussing: under which circumstances are we fine consuming third-party binaries, and if it's an option, should we plan to do more of it to lower the maintenance burden and build times? E.g, there are binary releases for the Rust compiler or Go, why not use them, too?

Reply:
Yes these are binary releases. Normally I prefer the build from source approach, but it wouldn't be possible to do that from the ebuild here:
> The execution environment must have access to a Docker daemon (all build operations are performed in Docker containers for isolation from the host system).
Though we could rebuild from source and host the binaries for ourselves, and then consume them.

> E.g, there are binary releases for the Rust compiler or Go, why not use them, too?
I would be for switching to dev-lang/rust-bin, the build times for it are ridiculous, and the rust compiler (including self bootstrap) is a major part of our SDK build time. For go we wouldn't have much benefit from using a pre-built toolchain.

Reply:
Also given that we can revisit the choice of where the python binaries come from when migrating to sysext, I'm fine consuming the published binaries in this case here.

About Rust, yeah, specially the LLVM compilation takes ages…

Reply:
Rust cross-compilation support is something we handle in dev-lang/rust and would need to be ported. I haven't looked deeper.

Reply:
Yeah, or use `rustup` to install the regular toolchain into the SDK